sandbox/group_replication.go

```diff
@@ -284,9 +284,9 @@ func CreateGroupReplication(sandboxDef SandboxDef, origin string, nodes int, mas
 		sandboxDef.NodeNum = i
 		// fmt.Printf("%#v\n",sdef)
 		logger.Printf("Create single sandbox for node %d\n", i)
-		err, execList := CreateChildSandbox(sandboxDef)
+		execList, err := CreateSingleConcurrentSandbox(sandboxDef)
```

Comment:
The name should be `ChildSandbox` (or `InnerSandbox`) rather than `SingleConcurrentSandbox` because the sandboxes deployed for multiple configurations can be deployed either sequentially (the default) or concurrently.